src/v2/schema/query.js

```diff
@@ -54,6 +54,9 @@ type Query {
   # Get secrets
   secrets(label: String, value: String): [Secret]
 
+  # Get secrets
+  managedCluster(clusterName: String): managedCluster
+
```

Comment:
Comment is copy-paste error. Please consider the name of the query. Does it indicate what it does? Would it be possible to just return a boolean rather than an object containing a boolean? Unless you think more fields will be added, probably not worth it.